### PR TITLE
Show supported apps in app management

### PR DESCRIPTION
**Pull request by MorrisJobke**
Based on #15495 

This shows supported apps in green and official apps in black:

Bildschirmfoto 2019-05-10 um 13 57 57

Does anybody want to compile it. Locally I can only build the development edition, because the SCSS compiler claims that macOS 64 bit is not supported and thus doesn't include them in the production bundles.

### CODE REVIEW

**Review by rullzer — APPROVED** [review]
Nice!

**Review by georgehrke — APPROVED** [review]

File: settings/css/settings.scss [hunk 1]
```diff
@@ -682,10 +682,16 @@ span.version {
...
+		background-position: left center;
+		background-position: 5px center;
+		padding-left: 25px;
+	}
+	.supported {
 		border-color: #37ce02;
 		background-position: left center;
 		background-position: 5px center;
 		padding-left: 25px;
+		color: #279001;
```
Comment (skjnldsv) [inline settings/css/settings.scss]: `var(--color-success)` ?

**Review by ChristophWurst — APPROVED** [review]

**Comment by skjnldsv** [comment]
Pushed and compiled

**Comment by skjnldsv** [comment]
Drone is not accepting the test :thinking:

**Comment by MorrisJobke** [comment]
> Drone is not accepting the test 🤔

Expected, because it is against another feature branch and not against master ;)

**Comment by skjnldsv** [comment]
:see_no_evil:

**Comment by MorrisJobke** [comment]
Updated to be against master.

**Review by skjnldsv — APPROVED** [review]

**Comment by MorrisJobke** [comment]
> The backport to stable16 failed. Please do this backport manually.

#15593

**Pull request merged**